File: Python/condicionais/conversao_temperatura.py
# Exercício: Conversão de temperatura
# Objetivo: converter uma temperatura entre as escalas Celsius e Fahrenheit.

unidade = input("Voce vai digitar a temperatura em qual escala (C/F)? ")

if unidade == "F":
    f = float(input("Digite a temperatura em Fahrenheit: "))
    c = 5 / 9 * (f - 32)
    print(f"Temperatura equivalente em Celsius: {c:.2f}") 
else:
    c = float(input("Digite a temperatura em Celsius: "))
    f = 9 * c / 5 + 32
    print(f"Temperatura equivalente em Fahrenheit: {f:.2f}")

File: Java/Estruturas Condicionais/conversao_temperatura.java
// Exercício: Conversão de temperatura
// Objetivo: converter uma temperatura de Fahrenheit para Celsius ou de Celsius para Fahrenheit.

import java.util.Locale;
import java.util.Scanner;

public class conversao_temperatura {
    public static void main(String[] args) {
        
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double C, F;
        char unidade;

        System.out.print("Voce vai digitar a temperatura em qual escala (C/F)? ");
        unidade = sc.nextLine().charAt(0);

        if (unidade == 'F') {
            System.out.print("Digite a temperatura em Fahrenheit: ");
            F = sc.nextDouble();
            C = 5.0 / 9.0 * (F - 32);
            System.out.println("Temperatura equivalente em Celsius: " + String.format("%.2f", C));
        }
        else {
            System.out.print("Digite a temperatura em Celsius: ");
            C = sc.nextDouble();
            F = 9.0 * C / 5.0 + 32;
            System.out.println("Temperatura equivalente em Fahrenheit " + String.format("%.2f", F));
        }

        sc.close();
    }
}